frontend/src/store/modules/LookupStore.ts

```diff
@@ -252,5 +263,11 @@ export default {
     sobonOrientierungswertJahr(context: ActionContext<LookupState, RootState>, list: LookupEntryDto[]): void {
       context.commit("sobonOrientierungswertJahr", list);
     },
+    sobonOrientierungswertJahrWithoutStandortabfrage(
```

Comment:
Da die meisten Variablen auch eher eine Zusammensetzung deutscher Wörter sind, würde ich das hier ebenfalls so machen.

Reply:
In den Coding-Guidelines ist definiert, dass fachliche Benennungen in Deutsch und der Rest in Englisch geschrieben wird.